libbpf-tools/tcppktlat_example.txt

```diff
@@ -39,27 +39,109 @@ PID     COMM             LADDR           LPORT RADDR           RPORT MS
+        64 -> 127        : 6        |****************************************|
+       128 -> 255        : 1        |******                                  |
+^C
+
+
+The -H option can also take an interval argument to print histograms periodically.
+
+# tcppktlat -H 5
+Tracing TCP packet latency. Hit Ctrl-C to end.
+
+pid = 1357503 verge-mihomo
+     usecs               : count    distribution
+         0 -> 1          : 1        |***                                     |
+         2 -> 3          : 1        |***                                     |
+         4 -> 7          : 4        |*************                           |
+         8 -> 15         : 4        |*************                           |
+        16 -> 31         : 7        |***********************                 |
+        32 -> 63         : 12       |****************************************|
+        64 -> 127        : 9        |******************************          |
+       128 -> 255        : 1        |***                                     |
+
+pid = 1357503 verge-mihomo
+     usecs               : count    distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 0        |                                        |
+         8 -> 15         : 2        |*************                           |
+        16 -> 31         : 2        |*************                           |
+        32 -> 63         : 1        |******                                  |
+        64 -> 127        : 6        |****************************************|
+       128 -> 255        : 1        |******                                  |
+^C
+
+
+The -p option can be combined with -H to show histogram for a specific PID only.
+
+# tcppktlat -p 1357503 -H
+Tracing TCP packet latency. Hit Ctrl-C to end.
+^C
+
+pid = 1357503 verge-mihomo
+     usecs               : count    distribution
+         0 -> 1          : 1        |***                                     |
+         2 -> 3          : 1        |***                                     |
+         4 -> 7          : 4        |*************                           |
+         8 -> 15         : 4        |*************                           |
+        16 -> 31         : 7        |***********************                 |
+        32 -> 63         : 12       |****************************************|
+        64 -> 127        : 9        |******************************          |
+       128 -> 255        : 1        |***                                     |
+^C
+
+
 # tcppktlat --help
 Usage: tcppktlat [OPTION...]
 Trace latency between TCP received pkt and picked up by userspace thread.
 
-USAGE: tcppktlat [--help] [-T] [-p PID] [-t TID] [-l LPORT] [-r RPORT] [-v]
+USAGE: tcppktlat [--help] [-T] [-H] [-p PID] [-t TID] [-l LPORT] [-r RPORT] [-w] [-v]
```

Comment:
The USAGE description in tcppktlat_example.txt is inconsistent with the actual implementation in tcppktlat. c